packages/api-server/src/api/portal.ts

```diff
@@ -102,4 +149,48 @@ router.post('/', async (req, res) => {
   }
 })
 
+router.delete('/', async (req, res) => {
+  try {
+    const portalIds = req.body.portals
+      .map((p: number) => {
+        const id = Number(p)
+
+        if (isNaN(id)) {
+          return null
+        }
+
+        return id
+      })
+      .filter(Boolean)
+
+    await deleteServerPortal(portalIds, req.userId, req.serverId)
+    res.send(204)
+  } catch (err) {
+    logger.log.error('Unable to delete', err)
+    res.status(500).send({ error: 'Error deleting portal' })
+  }
+})
+
+// TODO update in later PR
```

Comment:
What?

Reply:
I explained at the beginning of the PR.

Reply:
I see, remove this from these commits then and bring it in with the rest.